### PR TITLE
add: alittle more documentation

### PR DESCRIPTION
**Pull request by maxwelllwang**

### CODE REVIEW

File: src/brain/README.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
 # brain
 
-The brain contains modules that listen to webhook events from GitHub and Slack. These modules are autoloaded by `@utils/loadBrain`.
+The brain contains modules that listen to webhook events from GitHub, Slack, and GoCD. These modules are autoloaded by `@utils/loadBrain`.
```
Comment (mwarkentin) [inline src/brain/README.md:3]: Should we include options automator in this list too?

File: src/brain/README.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+
+(note) in order for load brain to load your function, it must be in a `index.ts` file and the name of the loading function must match that of your new directory. Ex: `gocdSlackFeeds/`, folder has `index.ts` with `export async function gocdSlackFeeds()`
+
+## Webhooks deployment
+
+* Webhooks in "production" are deployed to a Google Cloud Run instance, in the project `super-big-data`. Why? (TODO insert why)
```
Comment (mwarkentin) [inline src/brain/README.md:9]: Is this TODO something you want to do in this PR? Or later?

File: src/brain/README.md [hunk 3]
```diff
@@ -1,3 +1,18 @@
...
+* The webhook points to `https://product-eng-webhooks-vmrqv3f7nq-uw.a.run.app`
+
+## Adding a webhook to GoCD event emitter
+
+* goto [gocd](deploy.getsentry.net)
+* goto `admin` then `plugins`
+* find the `WebHook Notifier`
+* expand the text box by dragging the bottom right
+* Add your URL
```
Comment (mwarkentin) [inline src/brain/README.md:18]: Sorry, this was triggering me :D 

```suggestion
* add your URL
```

Comment (maxwelllwang) [reply src/brain/README.md:18]: Haha ok I'll work on consistency

File: src/brain/README.md (hunk 1 above)
Comment (maxwelllwang) [reply src/brain/README.md:3]: Yes good idea

Comment (maxwelllwang) [reply src/brain/README.md:3]: actually no options automator is a webhook but its not in the brain module.

**Review by mwarkentin — APPROVED** [review]

File: src/brain/README.md (hunk 1 above)
Comment (maxwelllwang) [reply src/brain/README.md:3]: actually no its not part of brain

**Pull request merged**